src/contrib/persistence/Akka.Persistence.Sqlite.Tests/Batching/BatchingSqliteAllEventsSpec.cs

```diff
@@ -22,14 +22,14 @@ public class BatchingSqliteAllEventsSpec : AllEventsSpec
         public static Config Config => ConfigurationFactory.ParseString($@"
             akka.loglevel = INFO
             akka.persistence.journal.plugin = ""akka.persistence.journal.sqlite""
+            akka.persistence.query.journal.sql.refresh-interval = 1s
```

Comment:
We have been using `refresh-interval` in Query unit tests wrong, `refresh-interval` is not part of the `journal` settings, but the `query` settings

Reply:
Are we going to need to update documentation anywhere for this? @eaba